book/website/collaboration/maintain-review.md

```diff
@@ -0,0 +1,28 @@
+(cl-maintain-review)=
+# Maintainers and Reviewers on GitHub
+
+## Prerequisites / recommended skill level
```

Comment:
```suggestion
## Prerequisites
```

Reply:
The recommended title for this section in the template is `Prerequisites` according to the chapter template

Reply:
Makes sense!